src-tauri/src/lib.rs

```diff
@@ -50,14 +51,49 @@ fn is_debug() -> bool {
 #[cfg_attr(mobile, tauri::mobile_entry_point)]
 pub fn run() {
     tauri::Builder::default()
+        .plugin(tauri_plugin_single_instance::init(|app, _args, _cwd| {
+            let _ = app
+                .get_webview_window("main")
+                .expect("no main window")
+                .set_focus();
```

Comment:
Is this a `Result`? Sometimes I do `.ok()` on results to discard their values if I don't care about them.